Runtime/Waypoint.cs

```diff
@@ -147,10 +150,10 @@ public static void Init(string waypointOrigin, string clientId, string deeplinkS
         public static string OnSignTypeData(string typedData, string from = null) => SignTypedData(typedData, from);
 
         [Obsolete("To be removed in 0.5.0. Use SendNativeToken() instead.")]
-        public static string OnSendTransaction(string receiverAddress, string value, string from = null) => SendNativeToken(receiverAddress, value, from);
+        public static string SendTransaction(string receiverAddress, string value, string from = null) => SendNativeToken(receiverAddress, value, from);
 
-        [Obsolete("To be removed in 0.5.0. Use SendTransaction() instead.")]
-        public static string OnCallContract(string contractAddress, string data, string value = "0x0", string from = null) => SendTransaction(contractAddress, data, value, from);
+        [Obsolete("To be removed in 0.5.0. Use WriteContract() instead.")]
+        public static string OnCallContract(string contractAddress, string data, string value = "0x0", string from = null) => WriteContract(contractAddress, data, value, from);
```

Comment:
WriteContract missing functionParameters?

Reply:
previous version OnCallContract required encoded function data! but this change need the raw data? I think it not backward compatible

Reply:
Fixed!